src/components/modals/SettingsModal.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+  <Modal @close="emitClose">
+    <div class="SettingsModal">
+      <div class="SettingsModal__header">
+        <h2 class="text-3xl mb-4">{{ $t("MODAL_SETTINGS.TITLE") }}</h2>
+        <p class="semibold text-grey mb-6">{{ $t("MODAL_SETTINGS.DESCRIPTION") }}</p>
+      </div>
+
+      <div v-if="isLoading" class="absolute inset-0 flex flex-col items-center justify-center z-10">
+        <Loader :data="null" />
+      </div>
+
+      <div class="SettingsModal__content">
+        <ListDivided>
+          <ListDividedItem
+            :label="$t('MODAL_SETTINGS.CURRENCY.LABEL')"
+            :label-description="$t('MODAL_SETTINGS.CURRENCY.DESCRIPTION')"
+            item-label-description-class="hidden md:block"
+          >
+            <InputSelect
+              :select-options="selectCurrencies"
+              :value="currencyName"
+              name="currency"
+              class="SettingsModal__inputSelect SettingsModal__select__currency"
+              @input="onSelectChange"
+            />
+          </ListDividedItem>
+          <ListDividedItem
+            :label="$t('MODAL_SETTINGS.DARK_THEME.LABEL')"
+            :label-description="$t('MODAL_SETTINGS.DARK_THEME.DESCRIPTION')"
+            item-label-description-class="hidden md:block"
+          >
+            <ButtonSwitch :is-active="nightMode" class="SettingsModal__toggle__darkTheme mt-2" @change="toggleTheme" />
+          </ListDividedItem>
+          <ListDividedItem
+            :label="$t('MODAL_SETTINGS.PRICE_CHART.LABEL')"
+            :label-description="$t('MODAL_SETTINGS.PRICE_CHART.DESCRIPTION')"
+            item-label-description-class="hidden md:block"
+          >
+            <ButtonSwitch :is-active="chartMode" class="SettingsModal__toggle__priceChart mt-2" @change="toggleChart" />
+          </ListDividedItem>
+          <ListDividedItem
+            :label="$t('MODAL_SETTINGS.LANGUAGE.LABEL')"
+            :label-description="$t('MODAL_SETTINGS.LANGUAGE.DESCRIPTION')"
```

Comment:
can you remove the descriptions? they aren't really needed for these simple settings

Reply:
@ItsANameToo just to confirm, remove all descriptions?

Reply:
correct